bitnami/dataplatform-bp1/Chart.yaml

```diff
@@ -19,20 +19,25 @@ dependencies:
     name: zookeeper
     repository: https://charts.bitnami.com/bitnami
     version: 6.x.x
+  - condition: wavefront.enabled
+    name: wavefront
+    repository: https://charts.bitnami.com/bitnami
   - name: common
     repository: https://charts.bitnami.com/bitnami
     tags:
       - bitnami-common
     version: 1.x.x
 description: OCTO Data platform Kafka-Spark-Solr Helm Chart
 engine: gotpl
-home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform
+home: https://github.com/bitnami/charts/tree/master/bitnami/dataplatform-bp1
```

Comment:
I think we can already add the icon link pointing to the bitnami one.

Reply:
can you please give me the link, will update